src/uosc/lib/menus.lua

```diff
@@ -148,10 +148,10 @@ function create_select_tracklist_type_menu_opener(menu_title, track_type, track_
 					title = (track.title and track.title or t('Track %s', track.id)),
 					hint = table.concat(hint_values, ', '),
 					value = track.id,
-					active = track.selected,
+					active = track.selected and track.id == tonumber(mp.get_property(track_prop)),
```

Comment:
I'm somewhat bothered by `tonumber(mp.get_property(track_prop))` being done twice on each iteration of the loop.

Youtube videos can add a ton of subtitle tracks if not restricted to specific languages via the ytdl options, and I don't know if the difference would be noticeable (probably not tbh), but since it's easy to avoid we might as well :smile:

Reply:
Change to use local variables, but only do it in a loop anyway.

Reply:
It's still doing `tonumber(mp.get_property(track_prop))` on each loop iteration though.

Reply:
I see what you mean, done.